doc/rst/platforms/cray.rst

```diff
@@ -210,7 +232,7 @@ Using Chapel on a Cray System
    See :ref:`readme-compiling` or  ``man chpl`` for further details.
 
 
-3) If ``CHPL_LAUNCHER`` is set to anything other than ``none``, when you
+4) If ``CHPL_LAUNCHER`` is set to anything other than ``none``, when you
```

Comment:
I wonder if we should just use automatic rst numbering (I think replace everything after `1)` with `#)`) instead of always having to update this. It's less nice when reading the raw file, but easier to maintain.